### PR TITLE
enhance sanity check for recent LibTIFF easyconfigs

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
cfr. #17974

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -37,11 +37,14 @@ dependencies = [
...
+configopts += "--disable-webp --disable-libdeflate "
 
 sanity_check_paths = {
-    'files': ['bin/tiffinfo'],
+    'files': ['bin/tiffdump', 'bin/tiffinfo', 'include/tiff.h', 'lib/libtiff.a', 'lib/libtiff.%s' % SHLIB_EXT,
+              'lib/libtiffxx.a', 'lib/libtiffxx.%s' % SHLIB_EXT, 'lib/pkgconfig/libtiff-4.pc'],
     'dirs': [],
 }
 
+sanity_check_commands = ["tiffinfo -h"]
```
Comment (Micket) [inline easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-10.2.0.eb:48]: -h flag doesn't exist for 4.1.0 it seems

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-10.2.0.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-10.2.0.eb]: I reverted the sanity check commands for you.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-10.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-10.2.0.eb]: Thanks @Micket!

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Pull request merged**